daemon/container_linux.go

```diff
@@ -763,31 +775,51 @@ func (container *Container) AllocateNetwork() error {
 		return nil
 	}
 
+	var networkDriver string
+	service := container.Config.PublishService
 	networkName := mode.NetworkName()
+	canCreateNetwork := false
```

Comment:
Not required. it was added for readability.
But have changed it (added comments instead).